whitepapers/business-value/Roadmap.md

```diff
@@ -0,0 +1,19 @@
+## OSS for Business Whitepaper Roadmap
+
+More Context for reviewing purposes.\
+Contributors working on the OSS in Business Whitepaper can use this roadmap that establishes an estimated timeline and track milestones achieved overtime. This roadmap is linked to the readme.md file.
+The roadmap file entails the following details in a structured format with the tag of the group leads responsible for specific sections.
```

Comment:
```suggestion
The roadmap file entails the following details in a structured format with the tag of the group leads responsible for specific sections.

|    | **Phase**                | **Timeline**          | **Tasks**                                                                                                                                                           |
|----|---------------------------|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| ✅ | Initial Discussions       | May - September 2024 | <ul><li>Conduct initial group meetings, discussions, and orientation during touchpoint calls.</li><li>Gather group feedback and gauge interest in participation.</li><li>Formalize the proposal and share it with the Steering Committee (SC) to request the creation of an official working group.</li></ul> |
| ✅ | Working Group Setup       | September - December 2024 | <ul><li>Establish the working group.</li><li>Define the whitepaper outline and set the overall mission statement.</li><li>Set up the working repository and communication channels.</li></ul> |
| 🗂️ | Chair Recruitment         | January - February 2025 | <ul><li>Recruit group chairs for each section.</li><li>For each section, chairs need to work with group participants to define the scope, milestones and preferred format for gathering and publishing content in the working repository.</li></ul> |
| 🗂️ | Monthly Progress          | March - September 2025 | <ul><li>Chairs present their group’s progress during monthly touchpoint meetings starting in March.</li><li>Document group progress and share updates with the TODO Steering Committee for review, guidance, and support as needed.</li><li>Include new terms introduced during content creation in the glossary.md file as they appear.</li></ul> |
| 🗂️ | Resource Aggregation      | September - October 2025 | <ul><li>Schedule a special working group meeting with all chairs and project managers to finalize resource aggregation and merge all sections into a comprehensive whitepaper.</li><li>Share a draft version of the whitepaper (not proofread) with the SC for final review.</li></ul> |
| 🗂️ | Proofreading & Design     | October - December 2025 | <ul><li>Transfer the whitepaper to a Google Doc for the LF staff to proofread and design, ensuring adherence to TODO brand guidelines.</li><li>LF staff coordinate the distribution of the final whitepaper PDF across LF communication channels.</li></ul> |

```

Reply:
@harmonyelendu  included some of the milestones that were already accomplished in 2024 (e.g working group formation, mission statement, etc)